packages/tour-kit/src/components/tour-kit-step.tsx

```diff
@@ -29,6 +34,19 @@ const TourKitStep: React.FunctionComponent< Props > = ( {
 		config.steps[ currentStepIndex ].options?.className
 	);
 
+	useEffect( () => {
```

Comment:
Thinking whether we can abstract this into a hook and avoid passing state through this way?

Reply:
Abstracted into `useStepTracking` in [d3309c7](https://github.com/Automattic/wp-calypso/pull/59719/commits/d3309c7f34e74ae97bdf1dcfad8f98bc772cbcd0)